pyaedt/edb_core/hfss.py

```diff
@@ -478,3 +479,143 @@ def create_hfss_ports_on_padstack(self, pinpos, portname=None):
             return True
         else:
             return False
+
+    @aedt_exception_handler
+    def create_lumped_port_on_trace(
```

Comment:
Maybe the name is not self-explanatory. If this methods created ports on traces on bounding of the layout only. Docstrings can help but I think could be helpful if name is intuitive